libs/EXTERNAL/libtatum/libtatum/tatum/TimingGraph.cpp

```diff
@@ -483,7 +483,7 @@ tatum::util::linear_map<EdgeId,EdgeId> TimingGraph::optimize_edge_layout() const
     //Determine the edges driven by each level of the graph
     std::vector<std::vector<EdgeId>> edge_levels;
     for(LevelId level_id : levels()) {
-        edge_levels.push_back(std::vector<EdgeId>());
+        edge_levels.emplace_back();
```

Comment:
Why do this instead of resizing the edge_levels array to levels().size() during construction?
```
std::vector<std::vector<EdgeId>> edge_levels(levels().size())
```